website/docs/reference/resource-configs/tags.md

```diff
@@ -153,16 +180,52 @@ models:
         - "published"
 
 ```
+</File>
+
+
+### Apply tags to models
+
+This section shares how to apply tags to models in the `dbt_project.yml`, `schema.yml`, or SQL files, however you can also apply tags to individual resources. 
```

Comment:
I'd leave that section out if we've removed that part of this section or link to wherever else it is
```suggestion
This section demonstrates applying tags to models in the `dbt_project.yml`, `schema.yml`, and SQL files. 
```